feat: audit-secureblue: add suggestions for new perm checks

spaceoden opened the pull request:
the newly added host-os:ro and device=all checks should offer suggestions as well

spaceoden committed:
feat: audit-secureblue: add suggestions for new perm checks

TheHooly commented:
I'd highly recommend adding device=dri when device=all is revoked. Otherwise many apps will use software rendering or stop working entirely.

spaceoden committed:
feat: add suggestion to enable device=dri along with nodevice=all 

many apps rely on device=all for gpu access. device=dri will allow them to retain it.

spaceoden committed:
feat: add "also" to suggestion for clarity

spaceoden commented:
@TheHooly I added a recommendation to enable device=dri

RoyalOughtness commented on files/justfiles/secureblue.just:
```diff
@@ -700,6 +702,11 @@ audit-secureblue:
                 warnings+=("> and introducing a vector for sandbox escapes!")
+                warnings+=("> To remove it use Flatseal or run:")
+                warnings+=("> 'flatpak override -u --nodevice=all $f'")
+                warnings+=("> We also recommend you enable device=dri to grant GPU access")
```
(nit) "to grant GPU access, if needed by the application"

spaceoden replied:
added, but said "app" instead of "application" to keep line width <80

spaceoden committed:
tweak: gpu suggestion

RoyalOughtness committed:
Merge branch 'live' into suggestions4newchecks
